libretro/libretro.cpp

```diff
@@ -201,50 +201,66 @@ static RetroOption<int> ppsspp_language("ppsspp_language", "Language", { { "auto
 static RetroOption<int> ppsspp_rendering_mode("ppsspp_rendering_mode", "Rendering Mode", { { "buffered", FB_BUFFERED_MODE }, { "nonbuffered", FB_NON_BUFFERED_MODE } });
```

Comment:
FYI calling it non-buffered is extremely misleading and confusing, because most users - familiar with other games and emulators - reasonably assume this is some setting to make the emulator double buffer to avoid shearing, or otherwise use extra buffering.  Someone looking to reduce input lag is going to think that "nonbuffered" is the Elite setting and "buffesed" is the Loser Hardware setting.

But that's wrong.

Technically, enabling this setting *does* enable double buffering, but only because PSP games *internally double buffer*.  When this speedhack is set to "nonbuffered", we force disable any usage of buffers by the game.  This means effects break, and sometimes rendering doesn't work at all.  Several popular games are totally broken when this unavoidably buggy speedhack is enabled.

We call it "skip buffer effects (speedhack)" in standalone to avoid this confusion.

-[Unknown]